test/main.cpp

```diff
@@ -1097,6 +1109,58 @@ int main() {
     inst = mavis_facade_rv32.makeInst(0x4041d213, 0);
     assert(inst != nullptr);
     cout << "line " << dec << __LINE__ << ": " << "DASM: 0x4041d213 = " << inst->dasmString() << endl;
+}
+
+    MavisType mav_andes({"../../json/isa_andestar.json"}, {}, uid_init, anno_overrides);
```

Comment:
You might want to construct mav_andes including some of the common RISC-V jsons, just to be sure there's no decoding conflict